data/389ds/tls_cert.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -euo pipefail
+
+CN_NAME=$1
+CA_DIR=$2
+rm -rf $CA_DIR
+mkdir -p $CA_DIR
+cd $CA_DIR
+openssl req -new -x509 -newkey rsa:2048 -keyout myca.key -days 3560 -out myca.pem -nodes -subj "/C=CN/ST=Beijing/L=Beijing/O=QA/OU=security/CN=389ds.example.com
+e.com"
+openssl genrsa -out server.key 2048
+openssl req -new -key server.key -out server.csr -subj "/C=CN/ST=Beijing/L=Beijing/O=QA/OU=security/CN=$1.example.com"
```

Comment:
Use the shell script to set up the configuration is rapid, and it looks can jump out of the setting interactive mode, do you think is there any side effect?

Reply:
hmm, nice comments, we can talk it offline to build a "package", but it is not an easy way

Reply:
Do you mean build a rpm package or ?  :)

Reply:
No, just a function. 
However, in that case, we should make sure it can fit other use cases. so just using a bash script here.

Reply:
Guys, no reason for bash scripts, neither for RPMs. Make a perl file under `lib/` and put your functions there so then you can reuse them from multiple test modules.

Reply:
I can't agree with you anymore, thanks all! let me do that